lizard_blockbox/views.py

```diff
@@ -1107,7 +1113,7 @@ def content_actions(self):
     @cached_property
     def breadcrumbs(self):
         result = super(AutomaticImportPage, self).breadcrumbs
-        result.append(Action(name=self.page_title))
+        result.append(Action(name=AutomaticImportPage.page_title))
```

Comment:
idem

Reply:
Ik fix de breadcrumbs in een aparte fix OK?